specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2245 @@
+import "@typespec/http";
+import "@typespec/versioning";
+import "@typespec/rest";
+import "./models.tsp";
+import "./main.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Traits.Private;
+using Azure.ClientGenerator.Core;
+
+namespace PurviewDataMap;
+
+model OkResponse<T> {
+  @statusCode
+  status: 200;
+
+  @body
+  data: T;
+}
+
+op AtlasImportOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse extends TypeSpec.Reflection.Model,
+  Traits extends TypeSpec.Reflection.Model = {},
+  TraitContexts extends TraitContext = TraitContext.Undefined
+> is Azure.Core.Foundations.Operation<
+  TParams &
+    TraitProperties<
+      Traits &
+        NoConditionalRequests &
+        NoRepeatableRequests &
+        NoClientRequestId,
+      TraitLocation.Parameters,
+      TraitContext.Action
+    >,
+  TResponse &
+    TraitProperties<
+      Traits &
+        NoConditionalRequests &
+        NoRepeatableRequests &
+        NoClientRequestId,
+      TraitLocation.Response,
+      TraitContext.Action
+    >,
+  Traits & NoConditionalRequests & NoRepeatableRequests & NoClientRequestId,
+  AtlasErrorResponse
+>;
+
+alias MultipartFormDataRequestHeadersTraits = Azure.Core.Traits.RequestHeadersTrait<{
+  @doc("The content type for the operation. Always multipart/form-data for this operation.")
+  @header("content-type")
+  contentType: "multipart/form-data";
+}>;
+
+interface Entity {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("""
+Create or update an entity.
+Existing entity is matched using its unique guid if
+supplied or by its unique attributes eg: qualifiedName.
+Map and array of
+collections are not well supported. E.g., array<array<int>>, array<map<string,
+int>>.
+For each contact type, the maximum number of contacts is 20.
+""")
+  @route("/atlas/v2/entity")
+  @post
+  createOrUpdate is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Used to define the update behavior for business attributes when updating
+entities.
+""")
+      @query
+      businessAttributeUpdateBehavior?: BusinessAttributeUpdateBehavior;
+
+      @doc("""
+The collection where entities will be moved to. Only specify a value if you
+need to move an entity to another collection.
+""")
+      @query
+      collectionId?: string;
+
+      @doc("Atlas entity with extended information.")
+      @body
+      body: AtlasEntityWithExtInfo;
+    },
+    EntityMutationResult,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/prefer-csv-collection-format" "This is an existing Atlas API"
+  @doc("List entities in bulk identified by its GUIDs.")
+  @route("/atlas/v2/entity/bulk")
+  @get
+  listByGuids is Azure.Core.Foundations.Operation<
```

Comment:
Does this operation return a paged response?

Reply:
No. This is also an Atlas API.